token-auth.yml

```
@@ -47,7 +47,17 @@ paths:
         tags:
         - Token Authentication
         summary: Create an Access Token
-        description: Create an Access Token
+        description: Create a token for a registered or an unregistered user. Pass the metnum_user_id for registered users.
```

```suggestion
        description: Create a token for a registered or an unregistered user. Pass the mentum_user_id for registered users.
```

Maybe we should also define what a `registered` and `unregistered` user is?